src/main/java/frc/robot/subsystems/TrapElvSubsystem.java

```diff
@@ -134,9 +139,15 @@ public TrapElvSubsystem() {
             TrapElvConstants.WRIST_PID[0],
             TrapElvConstants.WRIST_PID[1],
             TrapElvConstants.WRIST_PID[2]);
+    wristPIDController.setIZone(TrapElvConstants.WRIST_PID[3]);
+    wristFeedforward =
+        new ArmFeedforward(
+            TrapElvConstants.WRIST_FF[0],
+            TrapElvConstants.WRIST_FF[1] / 2.0,
```

Comment:
Don't think we meant to leave the divide by 2.0 here

Reply:
It does need to be half because it is still only one motor driving rotation and not two, but I will make this change in Constaints rather than here